### ISSUE TITLE
Automatically resolve NetworkManager/systemd-resolved MagicDNS warning

### ISSUE THREAD

**Issue by legowerewolf** [tailscale-dev/deck-tailscale#4]
Is [this fix](https://tailscale.com/kb/1188/linux-dns/#networkmanager--systemd-resolved) safe to do on the Deck? Can we detect the lack of symlink and fix it automatically?

**Comment by gustakasn0v** [comment]
I ran that fix in my Steam Deck a couple days ago, haven't had problems since.

The issue is how to do it in a way that survives SteamOS updates. I don't know a way to do that for changes to the `/etc` directory...

That being said, I had MagicDNS functionality in my CLI (didn't try web browsers or Game Mode) without the fix. I just did it because the warning shown when running `tailscale status` was affecting my work on [this PR](https://github.com/saumya-banthia/tailscale-control/pull/5)

**Comment by legowerewolf** [comment]
Yeah, this seems to be a low-priority thing. It might be something for actual Tailscalars to fix - if MagicDNS is working, then the check is wrong.

**Comment by ironicbadger** [comment]
Closed due to age.
